### DISCUSSION TITLE
Park UI latest update was a huge breaking change

### DISCUSSION THREAD

**Discussion by tyteen4a03** [cschroeter/park-ui#381] [General]
And was not communicated properly. I just found out all of my styles broke and I had to regenerate the components (which were placed in a *different* file structure) and now I'm having to fix import paths. 

This was not communicated in the changelog at all.

Not that I don't like the new import setup, but this should be in the changelog so I can anticipate the change.

**Comment by cschroeter** [comment]
@tyteen4a03 

Thanks for bringing this up. I agree that this was a significant change and it was not communicated properly. The changelog on the website is for @park-ui/panda-preset, so I wasn't sure where to document it. If you have any suggestions on how to handle this better in the future, I'd appreciate your input.

**Reply by tyteen4a03** [reply]
> @tyteen4a03
> 
> Thanks for bringing this up. I agree that this was a significant change and it was not communicated properly. The changelog on the website is for @park-ui/panda-preset, so I wasn't sure where to document it. If you have any suggestions on how to handle this better in the future, I'd appreciate your input.

As most people look at the GitHub Releases, putting a warning in the panda-preset package along with a document in the repo or on the website about migrating would be good.

**Comment by matthewtesimale** [comment]
@cschroeter I'd love to help maintain this project as it's a huge help in my work projects! Please let me know if there's anything I can do to help upkeep and maintain any upcoming changes to ParkUI! If you are running a discord, I can help facilitate announcements and such to provide better insight into future releases! Let me know if you'd like to collaborate and touch base on some opportunities! 

matthew.tesimale@gmail.com

Having a Migration.md file would always be good too!
